snapshot/lib/snapshot/fixes/hardware_keyboard_fix.rb

```diff
@@ -10,6 +10,9 @@ def self.patch
         UI.verbose("Patching simulator to work with secure text fields")
 
         Helper.backticks("defaults write com.apple.iphonesimulator ConnectHardwareKeyboard 0", print: FastlaneCore::Globals.verbose?)
+        
+        # For > Xcode 9
+        Helper.backticks("/usr/libexec/PlistBuddy -c \"Print :DevicePreferences\" ~/Library/Preferences/com.apple.iphonesimulator.plist | perl -lne 'print $1 if /^    (\\S*) =/' | while read -r a; do /usr/libexec/PlistBuddy -c \"Set :DevicePreferences:$a:ConnectHardwareKeyboard false\" ~/Library/Preferences/com.apple.iphonesimulator.plist || /usr/libexec/PlistBuddy -c  \"Add :DevicePreferences:$a:ConnectHardwareKeyboard bool false\" ~/Library/Preferences/com.apple.iphonesimulator.plist; done", print: FastlaneCore::Globals.verbose?)
```

Comment:
Can you add a comment above here with a link to a stackoverflow post or somewhere on where you got this from? (I'm assuming this is from a stackoverflow post) 😊